awx/main/migrations/0118_v400_add_remote_archive_scm_type.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.11 on 2020-08-18 22:35
```

Comment:
We recently stopped embedding downstream versions in these filenames - can you remove the `v400` ?

Reply:
I wasn't sure how to create the migration, since there doesn't seem to be any documentation about this in the contribution guide, so I was trying to match existing, but yes, I'll change it!